README.md

```diff
@@ -168,12 +168,36 @@ To delete a cluster, run:
 eksctl delete cluster --name=<name> [--region=<region>]
 ```
 
+### Creating nodegroup
```

Comment:
Probably better to have just one subheading on 'Managing nodegroups'

Reply:
Sounds good 👍

Reply:
Done in 4ebc4e7